.github/workflows/ci.yaml

```
@@ -31,52 +31,47 @@ jobs:
-
-      - name: Typecheck
-        run: bun turbo run typecheck
-
-      - name: Prettier --write
+      # Phase 2: Format (may commit changes, must run before lint)
+      - name: Format
         run: bun run format
 
-      - name: Commit Prettier changes if needed
+      - name: Commit formatting changes
         run: |
```

actionlint (1.7.9)

"github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)